client/app/components/ovh-angular-actions-menu/ovh-angular-actions-menu.module.js

```diff
@@ -1,5 +1,5 @@
 angular
   .module('App')
   .config((actionsMenuProvider) => {
-    actionsMenuProvider.setTranslationPath('bower_components/ovh-angular-actions-menu/dist/ovh-angular-actions-menu');
+    actionsMenuProvider.setTranslationPath('node_modules/ovh-angular-actions-menu/dist/ovh-angular-actions-menu');
```

Comment:
This one didn't need `../node_modules` ? Or it does the same thing than https://github.com/ovh-ux/ovh-manager-web/pull/408/files#diff-9a337108348a3685456acda14dad4ce2R3